src/index.js

```diff
@@ -193,6 +210,54 @@ class DropdownTreeSelect extends Component {
     this.keepDropdownActive = false
   }
 
+  onKeyboardKeyDown = e => {
+    const { readOnly, simpleSelect } = this.props
+    const { showDropdown, tags, searchModeOn, currentFocus } = this.state
+    const tm = this.treeManager
+    const tree = searchModeOn ? tm.matchTree : tm.tree
+
+    if (!showDropdown && (keyboardNavigation.isValidKey(e.key, false) || /^\w$/i.test(e.key))) {
+      // Triggers open of dropdown and retriggers event
+      e.persist()
+      this.handleClick(null, () => this.onKeyboardKeyDown(e))
+      if (/\w/i.test(e.key)) return
```

Comment:
`/\w/i.test(e.key)` is not Unicode safe. [This](https://mathiasbynens.be/notes/es6-unicode-regex#impact-character-class-escapes) makes my head spin, especially the note at the bottom. 

What is the intention here? If possible, we should look for non-regex based solutions.

Reply:
We return to avoid skipping (preventDefault) the event if it's a char-press.
Probably can be solved in other ways.